docs/Code Exposure.md

```diff
@@ -0,0 +1,261 @@
+
+1. Navigate to your [CloudTrail Dashboard](https://console.aws.amazon.com/cloudtrail)
+1. In the left-hand margin select `Event History`
+1. In the drop-down change from `Read-Only` to `Event Name`
+1. Review CloudTrail logs for the eventnames `GetObjectAcl` and `PutObjectAcl`
+
+### VPC Flow Logs
+VPC Flow Logs is a feature that enables you to capture information about the IP traffic going to and from network interfaces in your VPC. This can be useful for IP addresses discovered within CloudTrail to determine the types of external connections to any public resources. 
+
+For further information and steps, including querying with Athena, please refer to the [AWS Documentation for VPC Flow Logs](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs-athena.html). It is recommended that Athena analysis be included in a separate playbook and linked to other relevant items. 
+
+### Endpoint / Host Based
+1. Navigate to your [CloudTrail Dashboard](https://console.aws.amazon.com/cloudtrail)
+1. In the left-hand margin select `Event History`
+1. In the drop-down change from `Read-Only` to `Event Name`
+1. Review CloudTrail for `PutObject` and  `DeleteObject` requests from public IP addresses
+
+- Review EC2 operating system and application logs for inappropriate logins, installation of unknown software, or the presence of unrecognized files. 
+
+- It is highly recommended to have a third-party host-based intrusion detection system (HIDS) solution (such as OSSEC, Tripwire, Wazuh, [Amazon Inspector](https://aws.amazon.com/inspector/), other) 
+
+## Containment
+
+### S3 Block Public Access
+aws s3api put-public-access-block --bucket bucket-name-here --public-access-block-configuration "BlockPublicAcls=true,IgnorePublicAcls=true,BlockPublicPolicy=true,RestrictPublicBuckets=true"
+
+You can also review [Blocking public access to your Amazon S3 storage](https://aws.amazon.com/s3/features/block-public-access/) for additional details on blocking public S3 access across your account.
+
+## Eradication
+
+### S3 Remove Unrecognized / Unauthorized Objects
+Remove any unrecognized objects from buckets
+
+1. Sign in to the AWS Management Console and open the Amazon S3 console at https://console.aws.amazon.com/s3/
+1. In the Bucket name list, choose the name of the bucket that you want to delete an object from.
+1. Choose the name of the object that you want to delete.
+1. To delete the current version of the object, choose Latest version, and choose the trash can icon.
+1. To delete a previous version of the object, choose Latest version, and choose the trash can icon beside the version that you want to delete.
+
+## Recovery
+Same procedures as those listed for Eradication
+
+## Preventative Actions
+
```

Comment:
Any preventative actions for EC2 and CodeCommit?

Reply:
Added documentation for EC2 and CodeCommit.